geotrek/feedback/managers.py

```diff
@@ -1,7 +1,13 @@
 from django.contrib.gis.db import models
 
+from geotrek.common.mixins.managers import NoDeleteManager, TimestampedChoicesMixin
+
 
 class SelectableUserManager(models.Manager):
 
     def get_queryset(self):
         return super().get_queryset().filter(userprofile__isnull=False)
+
+
+class ReportManager(NoDeleteManager, TimestampedChoicesMixin):
```

Comment:
If you use NoDeleteMixin, you should use `existing` everywhere `objects` is used. In each queryset

Reply:
The manager for `Report` already was `NoDeleteManager` (implicit thanks to `NoDeleteMixin`), so `existing` is already called where needed (see `views.py`), I only added this here to extend it with my `TimestampedChoicesMixin` like I saw here : https://github.com/GeotrekCE/Geotrek-admin/blob/f1c4158565cf98929dc8cc039ff40a7c39f84399/geotrek/trekking/managers.py#L17